Xamarin/DrawXShared/RealmDraw.cs

```diff
@@ -338,7 +338,9 @@ private void DrawAPath(SKCanvas canvas, SKPaint paint, DrawPath drawPath)
         {
             using (var path = new SKPath())
             {
-                var pathColor = SwatchColor.ColorsByName[drawPath.color].Color;
+                // nasty hack because the Cocoa version defaults to Black
+                var pathColorName = (drawPath.color == "Black") ? "Charcoal" : drawPath.color;
```

Comment:
Generally, the right approach would be to store the color in an int, rather than by name, but that would require changing the model :/